modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/StorageEngineFactory.java

```diff
@@ -18,16 +18,26 @@
 package org.apache.ignite.internal.storage.engine;
 
 import java.nio.file.Path;
+import org.apache.ignite.configuration.schemas.store.DataStorageConfigurationSchema;
 import org.apache.ignite.internal.configuration.ConfigurationRegistry;
 import org.apache.ignite.internal.storage.StorageException;
 
 /**
  * Factory for creating storage engines.
  */
 public interface StorageEngineFactory {
+    /**
+     * Returns the unique name of the storage engine.
+     *
+     * <p>Used to map {@link DataStorageConfigurationSchema#name} to {@link StorageEngine}.
+     */
+    String name();
+
     /**
      * Creates a new storage engine.
      *
+     * <p>NOTE: {@code configRegistry} has not started yet, so only the root configuration can be retrieved.
```

Comment:
What does it even mean? Why can't you retrieve other configurations? They can be accessed, they just empty

Reply:
Rechecked, my comment is meaningless, deleted it